plugins/ocp/ocp-telemetry-decode.c

```diff
@@ -1204,10 +1204,14 @@ int parse_statistic(struct nvme_ocp_telemetry_statistic_descriptor *pstatistic_e
 		    struct json_object *pstats_array, FILE *fp)
 {
 	if (pstatistic_entry == NULL) {
-		nvme_show_error("Input buffer was NULL");
+		nvme_show_error("Statistics Input buffer was NULL");
 		return -1;
 	}
 
+	if (pstatistic_entry->statistic_id == STATISTICS_RESERVED_ID) {
+		return -2;
```

Comment:
Do these negative numbers have meaning? Then yous error codes instead, otherwise just return -1. Also braces are not necessary for single statement blocks.